src/SprykerEco/Zed/Payone/Business/TransactionStatus/TransactionStatusUpdateManager.php

```diff
@@ -1,4 +1,6 @@
 <?php
+// phpcs:disable Spryker.ControlStructures.DisallowCloakingCheck.FixableEmpty
+// phpcs:disable Spryker.ControlStructures.DisallowCloakingCheck.InvalidEmpty
```

Comment:
Why are those here? This should instead be fixed in code.

Reply:
Fixed.